packages/dx-grid-core/src/plugins/virtual-table-state/helpers.test.ts

```diff
@@ -272,6 +272,17 @@ describe('VirtualTableState helpers', () => {
           expect(calculateRequestedRange(virtualRows, newInterval, pageSize, referenceIndex, false))
           .toEqual({ start: 400, end: 500 });
         });
+
+        // T937684
+        it('should caclulate correct if infinite scrolling and end of calculated interval is not a multiple of the page size', () => {
```

Comment:
should calculate correctly if infinite scrolling is enabled and the end of the calculated interval is not divisible by the page size